examples/createtool/createtool.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>Cindy JS</title>
+        <style>
+            body {
+                margin: 0;
+                padding: 0;
+            }
+
+            button {
+                background-color: #FFFFFF;
+                border: 1px solid #CCCCCC;
+            }
```

Comment:
Actually I think it is possible to wrap image files like PNG within an SVG file. So that could be done in the interim. However it is probably not important if user interface entirely contained within this example and none of it ends up being entangled in the `CindyJS` library. To that end, the variable `activeButton;` and the functions `evaluator.createtool$3` and `addTool` belong within the UI on the outside of the `CindyJS` library. The `setActiveTool` is the function that needs to be exported for use by that UI.